### ISSUE TITLE
Datensatz- nur für Maike und Julia

### ISSUE THREAD

**Issue by JuNaPh** [hannahbartmn/WisA#2]
Kommuniaktion über Erstellung des Datensatzes
Nur für Maike und Julia

**Comment by MaikeBr0** [comment]
Hey, ich würde sagen, dass wir ID, Alter, Studienfach und Mathe LK einfach die Namen als Spaltennamen des Datensatzes beibehalten und halt bei Mathe LK, dass als Mathe_LK schreiben. Hast du Ideen für die anderen Variablen.

**Comment by JuNaPh** [comment]
Hi, das klingt gut. Für die Interessensvariablen würde ich Mathe_Int und Prog_Int vorschlagen.

**Comment by MaikeBr0** [comment]
Klingt gut, ich teile das den anderen mit

**Comment by MaikeBr0** [comment]
Ist das okay für dich, wenn ich die beiden Interessensvariablen machen, da diese sicher sehr ähnlich vom Aufbau sind. Ich denke das es vom Aufwand ähnlich sein wird, wie die anderen drei Variablen.

**Comment by JuNaPh** [comment]
Okay, dann schreibe ich die Simulationen für Alter, Studienfach und Mathe LK. Über die Wahrscheinlichkeiten können wir uns ja dann später austauschen.

**Comment by MaikeBr0** [comment]
Habe die set.seed() hinzugefügt

**Comment by JuNaPh** [comment]
Habe jetzt zu Studienfach und Mathe LK Wahrscheinlichkeiten eingefügt, was sagst du zu diesen? Hast du dir schon Wahrscheinlichkeiten für die Interessensvariablen überlegt? Ich würde vor allem zwischen Interesse an Mathematik und Studienfach Mathe beziehungsweise Interesse am Programmieren und Studienfach Informatik einen starken Zusammenhang simulieren.

**Comment by MaikeBr0** [comment]
Ich habe jetzt zu den beiden Interessensvariablen auch Wahrscheinlichkeiten hinzugefügt. Die kannst du dir mal anschauen und sagen, ob die so gut sind. Zu der Variable Mathe_LK finde ich, dass die Wahrscheinlichkeit für Interesse an Mathe größer als 4 etwas hoch. Klar werden sie sehr wahrscheinlich vorher Mathe LK gehabt haben, aber ich fände zB 0.75 ganz gut.

**Comment by JuNaPh** [comment]
Danke, habe das abgeändert und Kommentare zu deinen Wahrscheinlichkeitsangaben geschrieben, kannst du dir bei Gelegenheit mal durchlesen

**Comment by MaikeBr0** [comment]
Habe ich gemacht. Wollen wir den Datensatz bis Sonntag fertig stellen und als Datei im Main hochladen?

**Comment by JuNaPh** [comment]
Klingt gut, machen wir👍

**Comment by JuNaPh** [comment]
Zu den Wahrscheinlichkeiten der Interessensvariablen dachte ich mir noch, dass es plausibler wäre, für Statistik und Data Science  die Wkeit für Interesse an Mathe 1 oder 2 noch kleiner oder ganz auf null zu setzen. Auch bei Informatik und Interesse am Programmieren würde ich die Wkeiten für geringe Interessen niedriger stellen, was meinst du?

**Comment by MaikeBr0** [comment]
Ich habe die Wahrscheinlichkeiten noch mal abgeändert, kannst sie dir noch mal anschauen, ob die Werte so gut sind, oder ob sie doch noch anders sein sollten.

**Comment by MaikeBr0** [comment]
Habe die Wahrscheinlichkeit bei einigen nicht ganz Null gesetzt, um seltende Fälle nicht ganz auszuschließen. Was hälst du davon?

**Comment by JuNaPh** [comment]
Ja, finde die Wahrscheinlichkeiten so besser, habe auch noch die set.seeds in unterschiedliche Zahlen abgeändert. Dann ist der Datensatz soweit fertig? Wir können ja nächste Woche noch ein paar wenige Sachen abändern falls sich was ergibt.

**Comment by MaikeBr0** [comment]
Jap, der müsste so weit fertig sein.

**Comment by JuNaPh** [comment]
Ich habe jetzt mal die "Warnungen" am Anfang der Datensatz-Erstellungs-Datei weggemacht. Die anderen beiden können ja jetzt in diese Datei reinschauen und sich die Wahrscheinlichkeiten für die Simulation genauer anschauen, was meinst du?

**Comment by MaikeBr0** [comment]
Stimmt, damit ist es ja auch noch mal deutlicher, wie genau wir den Code geschrieben haben.
